dbms/src/Storages/KVStore/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ add_subdirectory (./MultiRaft/Disagg)
 
 add_library(kvstore ${kvstore_headers} ${kvstore_sources})
 target_include_directories(kvstore PRIVATE ${TiFlash_SOURCE_DIR}/contrib/GmSSL/include)
+target_include_directories(kvstore PUBLIC ${TiFlash_SOURCE_DIR}/contrib/not_null/include)
```

Comment:
I think `not_null` should be put to "common_io" and also move the file `Utils/NotNull.h` to `Common`

Reply:
OK